modules/kms/variables.tf

```diff
@@ -3,6 +3,18 @@ variable "region" {
   description = "AWS Region"
 }
 
+variable "alias" {
+  type        = string
+  description = "The display name of the alias. The name must start with the word alias followed by a forward slash. If not specified, the alias name will be auto-generated."
+  default     = null
+}
+
+variable "description" {
+  type        = string
+  description = "The description for the KMS Key."
+  default     = null
```

Comment:
If the description isn't set to the original value of `Parameter Store KMS master key`, won't that cause people's kms keys to be recreated?

Reply:
I'm not sure honestly as it has been a long time, but updated regardless 😅